Makefile

```diff
@@ -179,16 +179,13 @@ $(get-sources-tgt): build-info
 	@REPO=$(@:%.get-sources=$(SRC_DIR)/%) MAKE="$(MAKE)" $(BUILDER_DIR)/scripts/get-sources
 get-sources: $(get-sources-tgt)
 
-.PHONY: check-depend
-check-depend:
-	@if ! which rpm >/dev/null 2>&1; then
-		echo "WARNING: rpm executable not found (are you on cygwin?)"; \
-	elif [ $(VERBOSE) -gt 0 ]; then \
-		echo "currently installed dependencies:"; \
-		rpm -q $(DEPENDENCIES) || exit 1; \
-	else \
-		rpm -q $(DEPENDENCIES) >/dev/null 2>&1 || { echo "ERROR: call 'make install-deps' to install missing dependencies"; exit 1; }; \
-	fi
+.PHONY: check.rpm check-depend
+check.rpm: $(if $(shell which rpm 2>/dev/null), /bin/true, please.install.rpm.and.try.again);
```

Comment:
Any idea how to support also Debian host here? It would use `dpkg -l` instead of `rpm -q`.